tests/conftest.py

```diff
@@ -95,14 +102,21 @@ def _print_process_output(popen: subprocess.Popen) -> None:
 
 
 @fixture
-def application_process(in_container: bool, command_line: List):
+def check_app_exited() -> bool:
+    """Override this to prevent checking if app exited prematurely (useful for simulating crash)."""
+    return True
+
+
+@fixture
+def application_process(in_container: bool, command_line: List[str], check_app_exited: bool):
     if in_container:
         yield None
         return
     else:
         # run as non-root to catch permission errors, etc.
         def lower_privs():
             os.setgid(1000)
+            os.initgroups("ubuntu", 1000)
```

Comment:
What's that for

Reply:
ugh my pycharm is adding stuff to git unintentionally